airbase/pom.xml

```diff
@@ -216,7 +216,7 @@
         <dep.spotbugs-annotations.version>4.7.3</dep.spotbugs-annotations.version>
         <dep.testng.version>6.10</dep.testng.version>
         <dep.asm.version>9.5</dep.asm.version>
-        <dep.errorprone.version>2.19.1</dep.errorprone.version>
+        <dep.errorprone.version>2.20.</dep.errorprone.version>
```

Comment:
```suggestion
        <dep.errorprone.version>2.20.0</dep.errorprone.version>
```

Reply:
Yeah. My bad. Funny thing is that I've tested it and it actually works.

Reply:
Oh, I've fooled myself. I've made a change in one repo and tested on the other 🤦🏼